mdm/checkin_event.go

```diff
@@ -10,11 +11,12 @@ import (
 )
 
 type CheckinEvent struct {
-	ID      string
-	Time    time.Time
-	Command CheckinCommand
-	Params  map[string]string
-	Raw     []byte
+	ID         string
+	Time       time.Time
+	Command    CheckinCommand
+	Params     map[string]string
+	Raw        []byte
+	DeviceCert *x509.Certificate
```

Comment:
can this be `[]byte`? It will be easier if it doesn't have to be decoded/parsed every time.

Reply:
It can be. The idea was that perhaps we'd want to access it somehow for other processing/usage? But that's fine — no need to overcomplicate for a use case we don't yet have.